### ISSUE TITLE
can the multiple-streams and single-stream be merged into single Lambda function that maps to an Alexa skill with two intents

### ISSUE THREAD

**Issue by anu77777** [alexa/skill-sample-nodejs-audio-player#61]
is it possible to merge the multiple-streams and single-stream into a single Lambda function for an alexa skill with  two intents ? How can we achieve this ?

Any help is greatly appreciated

**Comment by sebsto** [comment]
Sure it can ! 

You'll need to merge the voice models and have be sure all intents are managed by the lambda function.  I would suggest to start by designing the voice experience you want to present to your customers.  What would be the dialog flow ?  The question asked by Alexa to perform the task.  Once your design is done, map it to a voice interaction model (intent and slots) and then code the intent in lambda.  

We can not provide one-to-one support, but you'll find many valuable advises at https://developer.amazon.com/alexa-skills-kit/design

Cheers
Have fun.
